src/main/java/no/nav/pto/veilarbportefolje/aktiviteter/AktivitetTyperFraKafka.java

```diff
@@ -0,0 +1,12 @@
+package no.nav.pto.veilarbportefolje.aktiviteter;
+
+public enum AktivitetTyperFraKafka {
```

Comment:
Enums are usually in capital letters. https://stackoverflow.com/questions/3069743/coding-conventions-naming-enums. Its not big fail, just something that can improve readability.

Reply:
I see that its the same thing in AktivitetTyperFraAktivitetspan, so I am not sure is it worth to change that...maybe it will only cause some troubles at the moment.